docs/topics_5/web-adding-global-custom-rules.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='adding_global_custom_rules_{context}']
+= Adding global custom rules
+
+The {ProductShortName} includes a preconfigured set of global rules, which apply to all projects.
+
+You can define your own custom global rules.
+
+For information on writing custom {ProductShortName} rules, see the {ProductShortName} link:{ProductDocRulesGuideURL}[_{RulesDevBookName}_].
+
+.Procedure
+
+. In the {WebName}, click *Rules configuration*.
++
+image::web-rules-config_51.png[Rules configuration screen]
```

Comment:
On second thought, take this image out. It is too tightly cropped. Doesn't show nav bar or "Add rules" button, so it's not useful for procedure. Also, it shows system rules, not custom rules.